services/catarse/catarse.js/legacy/src/root/admin-balance-tranfers.js

```diff
@@ -345,9 +355,10 @@ const adminBalanceTranfers = {
                 displayModal: state.displayApprovalModal,
                 content: state.generateWrapperModal({
                     modalTitle: 'Aprovar saques',
-                    ctaText: 'Aprovar',
+                    ctaText: 'Criar Lote',
```

Comment:
Tinha que alterar o Copy aqui para "Criar Lote na Transfeera", mas como quebrou o layout com o texto grande, coloquei ele menor para não precisar altera o componente. @thiagocatarse se for mandatório o copy aqui, eu modifico, só fui pelo caminho mais simples mesmo.

Reply:
de boa @grpse 👍